src/ext/java/org/opentripplanner/ext/siri/updater/SiriETHttpTripUpdateSource.java

```diff
@@ -15,7 +13,7 @@
 import java.util.Map;
 import java.util.UUID;
 
-public class SiriETHttpTripUpdateSource implements EstimatedTimetableSource, JsonConfigurable {
+public class SiriETHttpTripUpdateSource implements EstimatedTimetableSource {
```

Comment:
What exactly does class do? Interesting to know

Reply:
that's another shit code from OTP. Over design for 'Inheritance reuse'.